forgae-shape-vue-command.md

```diff
@@ -125,95 +125,114 @@ As a result, we can see a table with the most important details. For example:
+Assign the deployed contract address to the contractAddress property in ```aepp/src/contractDetails.js``` file.
 
-## Run the aepp
-Now we are ready to run the aepp.
+## Shape project
+
+The shape project is a sample Wallet/Identity Aepp that expects an Aepp to be loaded into an iFrame contained into this base aepp. 
+The most appropriate way to provide identity is the integration between the [base-aepp](https://base.aepps.com/) and your Æpp.
+The alternative identity provider is the custom identity aepp. We will use it just for simplicity and showing the app development process.
 
+## Run the identity/wallet provider aepp
+Let's start the identity/wallet Aepp, which will start on port 8080:
 ```
-cd aepp-forgae-shape-vue
+cd aepp-forgae-shape-vue/identity-provider
 npm run serve
 ```
-*By default the front-end app running at: Local: http://localhost:8080/*
 
-## Important considerations
-
-### Configurations
-The configuration file about contract details.
+## Run the aepp
+Now we are ready to run the aepp, which will start on port 8081. Open new terminal window and execute the following commands:
 
 ```
-../vue-shape/aepp-forgae-shape-vue/src/contractDetails.js
+cd aepp-forgae-shape-vue/aepp
+npm run serve
 ```
-We should provide the deployed contract address and the source code of the contract.
 
-The general configuration file.
-```
-../vue-shape/aepp-forgae-shape-vue/src/setting.js
-```
-In this file, we have to fill in the account private/public key pair and the network details.
+## Shape Project appearance
 
-### Get client
-The application flow starts with getting the client and contract instances. The code placed in the ```created``` lifecycle hook takes care of this as it is based on data from the configuration files.
+![shape project](https://raw.githubusercontent.com/aeternity/aepp-forgae-shape-vue/master/shape-project.png "Shape project")
 
-```javascript=
-async getClient() {
-    try {
-        const networkId = settings.networkId;
 
-        const clientNative = await Ae.compose({
-            props: {
-                url: settings.url,
-                internalUrl: settings.internalUrl,
-                compilerUrl: compilerUrl
-            }
-        })({ nativeMode: true, networkId })
+## Important considerations
 
-        const account = { secretKey: settings.account.priv, publicKey: settings.account.pub }
 
-        await clientNative.setKeypair(account)
+### Identity provider
 
-        this.client = clientNative
+#### Get client
+The application flow starts with getting the client. The code placed in the ```created``` lifecycle hook takes care of this.
 
-        this.$store.dispatch('setAccount', this.client);
-        this.accountBalance();
-        this.contractInstance = await this.client.getContractInstance(contractDetails.contractSource, { contractAddress: contractDetails.contractAddress });
-    } catch (err) {
-        console.log(err);
-    }
+```javascript=
```

Comment:
what does all of this has to do here? the an aepp developer should not care about this

Reply:
this question remains @VladislavIvanov